app/src/main/java/com/ifmo/balda/adapter/BoardGridAdapter.kt

```diff
@@ -114,7 +115,7 @@ class BoardGridAdapter(
   private fun getCurrentWord() = currentWord.fold("") { acc, it -> acc + it.first.text }
 
   private fun isCurrentWordValid(): Boolean {
-    return getCurrentWord() == "AAA"
+    return currentWord.map { (it.second / n) to (it.second % n) } in wordPositions
```

Comment:
В связи с тем, что я вмерджила свои изменения в мейн
```suggestion
    return currentWord.map { (it.position / n) to (it.position % n) } in wordPositions
```